pillar.example

```diff
@@ -1,8 +1,8 @@
 collectd:
   FQDNLookup: true
   TypesDB: ['/usr/share/collectd/types.db']
+  default_plugins: [battery, cpu, entropy, load, memory, swap, users]
```

Comment:
I don't like this change. Plugin related pillar config should go under ``plugins`` pillar. It was correct earlier